node-operators/networks/collator.md

```diff
@@ -33,19 +33,23 @@ Moonbeam使用[Nimbus平行链共识框架](/learn/features/consensus/)，通过
 
 ## 账户与质押要求 {: #accounts-and-staking-requirements }
 
-和波卡（Polkadot）验证人相似，收集人也需要创建账户。Moonbeam使用的是拥有私钥的H160账户或以太坊式账户。另外，需要拥有最低Token质押量才有资格成为候选人。只有一定数量的根据提名质押量排名靠前的收集人才会进入收集人有效集。
+和波卡（Polkadot）验证人相似，收集人也需要创建账户。Moonbeam使用的是拥有私钥的H160账户或以太坊式账户。另外，需要拥有高于最低自质押量(Self-bonded)才有资格成为候选人。只有一定数量的根据质押量和委托量排名靠前的收集人才会进入收集人有效集。
 
 === "Moonriver"
     |    变量     |                           值                           |
     |:---------------:|:---------------------------------------------------------:|
-    |   绑定数量   | {{ networks.moonriver.staking.min_can_stk }}枚MOVR  |
-    | 有效集上限 | {{ networks.moonriver.staking.max_candidates }}名收集人 |
+    |   绑定数量   | {{ networks.moonriver.staking.min_can_stk }} MOVR  |
+    | 最小自质押量 |     {{ networks.moonriver.staking.min_can_stk }} MOVR     |
```

Comment:
looks like these use the same variables, only one is needed for Minimum self-bond amount